new-lamassu-admin/src/pages/Customers/components/EditableCard.js

```diff
@@ -208,16 +250,26 @@ const EditableCard = ({
                   {!editing && (
                     <div className={classes.editButton}>
                       <div className={classes.deleteButton}>
-                        <ActionButton
-                          color="primary"
-                          type="button"
-                          Icon={DeleteIcon}
-                          InverseIcon={DeleteReversedIcon}
-                          onClick={() => deleteEditedData()}>
-                          {`Delete`}
-                        </ActionButton>
+                        {hasAdditionalData ? (
+                          <ActionButton
+                            color="primary"
+                            type="button"
+                            Icon={DeleteIcon}
+                            InverseIcon={DeleteReversedIcon}
+                            onClick={() => deleteEditedData()}>
+                            {`Delete`}
```

Comment:
I don't know if this this was deliberate choice, but this could be simpler.
```suggestion
                            Delete
```
(There are similar lines in this file)